degirum_tools/classification_eval.py

```diff
@@ -0,0 +1,145 @@
+import yaml, os
+from tqdm import tqdm
+from pathlib import Path
+
+
+def count_files_in_directory(directory_path):
+    # Ensure the directory path exists
+    if not os.path.exists(directory_path):
+        print(f"The directory path '{directory_path}' does not exist.")
+        return -1
+    # List all files in the directory
+    files = [
+        f
+        for f in os.listdir(directory_path)
+        if os.path.isfile(os.path.join(directory_path, f))
+    ]
+    # Count the number of files
+    file_count = len(files)
+    return file_count
+
+
+def model_prediction(model, validation_images_dir, label, labelsmap, k=1):
```

Comment:
label, labelsmap should be changed to category_folder and foldermap.
This function is just going to give you top k.
What if we want to should top 1 and top 5.